content/doc/book/managing/system-configuration.adoc

```diff
@@ -61,10 +59,10 @@ JENKINS_HOME
      +- [PLUGIN].jpi   (.jpi or .hpi file for the plugin)
  +- secret.key         (deprecated key used for some plugins' secure operations)
  +- secret.key.not-so-secret  (used for validating _$JENKINS_HOME_ creation date)
- +- secrets            (root directory for the secret+key for credential decryption)
-     +- hudson.util.Secret   (used for encrypting some Jenkins data)
+ +- secrets        (root directory for the secret+key for credential decryption)
      +- master.key           (used for encrypting the hudson.util.Secret key)
-     +- InstanceIdentity.KEY (used to identity this instance)
```

Comment:
In my test installation, this file is now named org.jenkinsci.main.modules.instance_identity.InstanceIdentity.KEY.  I'd rather have it renamed instead of being removed.

Reply:
If it is not present on your installation, that likely means you haven't installed some of the most common plugins like the Folders plugin, the Pipeline plugins, and the Git plugin.